manifests/o/oleg-shilo/cs-script/4.8.9.0/oleg-shilo.cs-script.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: oleg-shilo.cs-script
+PackageVersion: 4.8.9.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: cscs.exe
+  PortableCommandAlias: cscs
```

```suggestion
  PortableCommandAlias: cscs
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.SDK.8
```
Apologies, I'm getting ahead of myself.